src/viam/sdk/tests/test_mlmodel.cpp

```diff
@@ -12,17 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <viam/sdk/services/mlmodel/mlmodel.hpp>
+#include <viam/sdk/services/mlmodel.hpp>
 
 #include <memory>
 #include <tuple>
 #include <unordered_map>
 
 #include <boost/variant/get.hpp>
 
-#include <viam/sdk/services/mlmodel/client.hpp>
-#include <viam/sdk/services/mlmodel/private/proto.hpp>
-#include <viam/sdk/services/mlmodel/server.hpp>
+#include <viam/sdk/services/private/proto.hpp>
```

Comment:
This needs some thought: this test currently tests directly some internal proto to vocab conversions. I do want to keep testing that code as it is particularly tricky. I'll give it some thought, as there will probably be other places we need something like this.

Reply:
The tests aren't part of the `libviamsdk` library. Eventually, the types and symbols under `private` won't even be exported from `libviamsdk`, so the tests won't be able to use them. Better to clean it up now.

Reply:
Got it, thanks for clarifying.